### PR TITLE
Lesson02 

### PR DESCRIPTION
**Pull request by vad32m**
"Rock-scissors-paper" game implementation in C

### CODE REVIEW

File: 02-Git/solution/game.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+
+int main() {
+	printf("Not implemented yet\n");
+	return 0;
+}
```
Comment (an1kh) [inline 02-Git/solution/game.c]: Missing empty line

File: 02-Git/solution/game.c [hunk 2]
```diff
@@ -24,27 +27,43 @@ enum game_element char_to_element(char c)
 	case 'p':
 		return PAPER;
 	default:
-	        return INVAL_ELEM;
+		return INVAL_ELEM;
```
Comment (an1kh) [inline 02-Git/solution/game.c]: Fix original commit

File: 02-Git/solution/game.c [hunk 3]
```diff
@@ -8,13 +10,14 @@ enum game_element {
 	INVAL_ELEM
 };
 
-const char *element_to_str(enum game_element e) 
+
+const char *element_to_str(enum game_element e)
```
Comment (an1kh) [inline 02-Git/solution/game.c]: Extra line, don't see what else was changed here

**Comment by vad32m** [comment]
Fixed minor git usage issues, commit history may be reviewed one more time.

**Pull request merged**